polyfill/lib/calendar.mjs

```diff
@@ -184,6 +187,9 @@ impl['iso8601'] = {
   fields(fields) {
     return fields;
   },
+  mergeFields(fields, additionalFields) {
+    return ObjectAssign(fields, additionalFields);
+  },
```

Comment:
Will `Object.Assign` being the `ES.CalendarMergeFields` default if `mergeFields` is undefined make this code unnecessary? I think it can be omitted. Or is `mergeFields` only undefined if the prototype is overridden?

Reply:
This should be here nonetheless so that other calendars can chain up to it.